client/rpc/debug/src/lib.rs

```diff
@@ -712,7 +713,7 @@ where
 			let transactions = block.transactions;
 			if let Some(transaction) = transactions.get(index) {
 				let f = || -> RpcResult<_> {
-					let result = if trace_api_version >= 5 {
+					let result = if trace_api_version >= 7 {
```

Comment:
We need to have the save behaviour for version 5 and 6, otherwise we will add regressions to the tracing RPCs.

Reply:
Fixed in 72567438d840c8b369cacc8c63348a99d30e7133